### PR TITLE
Remove bcd from acookbook page

### PR DESCRIPTION
**Pull request by estelle**
All the features are well-supported and there's no mention of them not being supported, so I removed the compact tables as they are more than 50% of the page but don't add any info.

### CODE REVIEW

**Comment by wbamberg** [comment]
We don't omit compat for (say) CSS property reference pages when the feature is well-supported, so I'm not sure why we should do it here. We could decide across the board not to include compat for well-supported features (say, baseline features) but it would be better for that to be a consistent policy that's explicitly agreed, and I would not be in favour of it, personally.

File: files/en-us/web/css/layout_cookbook/center_an_element/index.md [hunk 1]
```diff
@@ -2,9 +2,6 @@
 title: Center an element
 slug: Web/CSS/Layout_cookbook/Center_an_element
 page-type: guide
-browser-compat:
-  - css.properties.align-items
-  - css.properties.justify-content
```
Comment (estelle) [inline files/en-us/web/css/layout_cookbook/center_an_element/index.md:7]:
```suggestion
browser-compat:
  - css.properties.flex
```

File: files/en-us/web/css/layout_cookbook/center_an_element/index.md [hunk 2]
```diff
@@ -94,10 +91,6 @@ div {
 
 Instead of applying `place-items: center;` on the container, you can achieve the same centering by setting {{cssxref("place-content", "place-content: center;")}} on the container or by applying either {{cssxref("place-self", "place-self: center")}} or {{cssxref("margin", "margin: auto;")}} on the inner item itself.
 
-## Browser compatibility
-
-{{Compat}}
-
```
Comment (estelle) [inline files/en-us/web/css/layout_cookbook/center_an_element/index.md:100]:
```suggestion
## Browser compatibility

{{Compat}}

```

**Review by estelle — COMMENTED** [review]
This is a cookbook page. In https://github.com/mdn/content/pull/29609 I removed the language that says "when alignment is supported...." i forgot to remove the bcd, but feel i should . The cookbook includes bcd b/c when it was written the modules discussed were not well supported. If we list BCD, we would need to list the 8 properties discussed in this rewrite. 

I feel this PR can be approved as is because the BCD is no longer relevant. however, if we must include a BCD to match the other pages which all need to be updated, instead of including bcd for the 8 properties (including `display` and `margin`, my suggested edits are to include `flex`, which isn't a property listed on the page, but looks like it means flex layout, so at least it is useful and relevant to the content.

**Comment by wbamberg** [comment]
> This is a cookbook page. In #29609 I removed the language that says "when alignment is supported...." i forgot to remove the bcd, but feel i should . The cookbook includes bcd b/c when it was written the modules discussed were not well supported. If we list BCD, we would need to list the 8 properties discussed in this rewrite.

I know it is a cookbook page, sorry if I was unclear about the "CSS property" comment. It was meant as an analogy. We currently include BCD for all cookbook pages. I think we should either include it for all cookbook pages, or for none, as we do for other page types (such as CSS properties).

As a rule across MDN, we don't have a policy of "omit BCD for well supported features" - so we don't do this for JS, or CSS reference, of Web/API etc. So if we have such a policy just for this one corner of MDN, it seems very likely to confuse people.

FWIW I would be fine to omit it for all cookbook pages. But we should be consistent (and what we choose should be the outcome of a discussion, not just a PR).

**Review by chrisdavidmills — APPROVED** [review]
FWIW, I agree with @estelle that the cookbook pages don't need compat tables. For me, the key points are:

- These are not reference pages, these are cookbook pages, which are more like guide pages really. And guide pages don't consistently have compat tables.
- They cover a combination of features, and listing all the compat tables for those features takes a lot of space. It isn't even beneficial - parsing a large swathe of compat tables isn't easy; it can be confusing.
- If you want to check support for those features, you can just click through to their reference pages.

So, I am approving this, but with a note that I think @wbamberg is right that we should have some kind of discussion first about how to handle this work for all the cookbook pages. For example:

- Do people agree that we can remove compat tables from all cookbook pages? (I'd second that)
- Do we need to include a consistent list of all the properties/features used in a recipe at the top, so it is easy to find further details on each feature if you want them?
- To make it clear that everything in the recipe is widely supported, should we make sure that the baseline ribbon is included at the top (like it is on https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Column_layouts), or include an experimental features banner of some kind in any cases where some things are not widely supported?

Once we have a consensus, I would be happy to help with the implementation.

**Comment by wbamberg** [comment]
> FWIW, I agree with @estelle that the cookbook pages don't need compat tables. For me, the key points are:
> 
>     * These are not reference pages, these are cookbook pages, which are more like guide pages really. And guide pages don't consistently have compat tables.
> 
>     * They cover a combination of features, and listing all the compat tables for those features takes a lot of space. It isn't even beneficial - parsing a large swathe of compat tables isn't easy; it can be confusing.
> 
>     * If you want to check support for those features, you can just click through to their reference pages.
> 
> 
> So, I am approving this, but with a note that I think @wbamberg is right that we should have some kind of discussion first about how to handle this work for all the cookbook pages. For example:
> 
>     * Do people agree that we can remove compat tables from all cookbook pages? (I'd second that)
> 
>     * Do we need to include a consistent list of all the properties/features used in a recipe at the top, so it is easy to find further details on each feature if you want them?
>

It sounds like the vehicle for this discussion should be a PR to add a template for CSS recipes to https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types .

(I agree with both your suggestions, FWIW. I just want us to do things consistently and to be deliberate in our choices.)

>     * To make it clear that everything in the recipe is widely supported, should we make sure that the baseline ribbon is included at the top (like it is on https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Column_layouts), or include an experimental features banner of some kind in any cases where some things are not widely supported?
> 

That's an interesting idea too. I was just moaning about how the baseline ribbon appears on so few pages that it doesn't seem very useful at the moment. Florian assures me that this situation will improve eventually :).

**Comment by chrisdavidmills** [comment]
> It sounds like the vehicle for this discussion should be a PR to add a template for CSS recipes to https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Page_types .

Sounds good to me.

> (I agree with both your suggestions, FWIW. I just want us to do things consistently and to be deliberate in our choices.)

Right on.

**Comment by estelle** [comment]
i'll merge this, then get to work on cookbook

**Pull request merged**